app/javascript/packs/MainApp/components/RefereeProfile.jsx

```diff
@@ -7,50 +7,151 @@ class RefereeProfile extends Component {
     match: PropTypes.shape({
       params: PropTypes.object
     }).isRequired
-  }
+  };
 
   state = {
+    httpStatus: 200,
+    httpStatusText: '',
     firstName: '',
     lastName: '',
     bio: '',
-    email: ''
-  }
+    email: '',
```

Comment:
As the data shape grows and we add update functionality we'll need to store more information in state. Maybe we could encapsulate the 'initial data' as it's own state object, so state could look like `{ httpStatus: 200, httpStatusTest: '', initialData: { firstName: '', ... }}`? Thoughts?

Reply:
I’ll think about this in the PR about profile editing.